kuma/api/v1/plus.py

```diff
@@ -62,3 +64,17 @@ def landing_page_survey(request):
         context["csrfmiddlewaretoken"] = get_token(request)
 
     return JsonResponse(context)
+
+
+@ratelimit(key="user_or_ip", rate="100/m", block=True)
+def landing_page_variant(request):
```

Comment:
I would add the following:

```suggestion
@require_GET
def landing_page_variant(request):
```